stdlib/public/core/Int128.swift

```diff
@@ -334,59 +402,90 @@ extension Int128: SignedNumeric {
 // MARK: - BinaryInteger conformance
 @available(SwiftStdlib 6.0, *)
 extension Int128: BinaryInteger {
-  
-  @inlinable
+  @available(SwiftStdlib 6.0, *)
+  @_transparent
   public var words: UInt128.Words {
     Words(_value: UInt128(_value))
   }
-  
+
+  @available(SwiftStdlib 6.0, *)
+  @_transparent
   public static func &=(a: inout Self, b: Self) {
     a._value = Builtin.and_Int128(a._value, b._value)
   }
-  
+
+  @available(SwiftStdlib 6.0, *)
+  @_transparent
   public static func |=(a: inout Self, b: Self) {
     a._value = Builtin.or_Int128(a._value, b._value)
   }
-  
+
+  @available(SwiftStdlib 6.0, *)
+  @_transparent
   public static func ^=(a: inout Self, b: Self) {
     a._value = Builtin.xor_Int128(a._value, b._value)
   }
-  
+
+  @available(SwiftStdlib 6.0, *)
+  @_transparent
   public static func &>>=(a: inout Self, b: Self) {
     let masked = b & 127
     a._value = Builtin.ashr_Int128(a._value, masked._value)
   }
-  
+
+  @available(SwiftStdlib 6.0, *)
+  @_transparent
   public static func &<<=(a: inout Self, b: Self) {
     let masked = b & 127
     a._value = Builtin.shl_Int128(a._value, masked._value)
   }
-  
+
+  @available(SwiftStdlib 6.0, *)
+  @_transparent
   public var trailingZeroBitCount: Int {
     _low == 0 ? 64 + _high.trailingZeroBitCount : _low.trailingZeroBitCount
   }
+
+  @available(SwiftStdlib 6.0, *)
+  @_transparent
+  public var _lowWord: UInt {
+#if _pointerBitWidth(_64)
+    UInt(_low._value)
+#elseif _pointerBitWidth(_32)
+    UInt(Builtin.trunc_Int64_Int32(_low._value))
+#else
+    UInt(truncatingIfNeeded: _low)
```

Comment:
Formally I think the `#else` branch here should probably hit the default implementation instead, but that requires refactoring this a bit, so I'm ok with this for now.

Reply:
Yeah this implementation is technically incorrect for words larger than 64 bits, so I just wrote this as truncating self.